extensions/amp-story-captions/0.1/amp-story-captions-presets.css

```diff
@@ -1,5 +1,18 @@
 @import '../../amp-story/1.0/amp-story-shadow-reset.css';
 
+/* 
+  auto append
+  Used when captions are provided from cache.
+*/
+.amp-story-captions-auto-append {
```

Comment:
Can publishers override this default position? If so, what CSS can they write to make it happen?

Reply:
We can always open for position customization later, starting without it sgtm

Also, please prefix the class name with `i-amphtml` :))